README.md

```diff
@@ -326,6 +328,80 @@ class PresentationViewController: UIViewController {
 
 You can learn more about `EpoxyPresentations` in its [wiki entry](https://github.com/airbnb/epoxy-ios/wiki/EpoxyPresentations).
 
+## EpoxyLayoutGroups
+
+LayoutGroups are UIKit [Auto Layout](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/AutolayoutPG/index.html) containers inspired by SwiftUI's [`HStack`](https://developer.apple.com/documentation/swiftui/hstack) and [`VStack`](https://developer.apple.com/documentation/swiftui/vstack) that allow you to easily compose UIKit elements into horizontal and vertical groups.
+
+`VGroup` allows you to group components together vertically to create stacked components like this:
+
+| ActionRow |
+| --------- |
+| ![ActionRow screenshot](docs/images/ActionRow.png) |
+
+```swift
+// Set of dataIDs to have consistent and unique IDs
+enum DataID {
+  case title
+  case subtitle
+  case action
+}
+
+// Groups are created declaratively just like Epoxy ItemModels
+let group = VGroup(alignment: .leading, spacing: 8) {
+  Label.groupItem(
+    dataID: DataID.title,
+    content: "Title text",
+    style: .title)
+  Label.groupItem(
+    dataID: DataID.subtitle,
+    content: "Subtitle text",
+    style: .subtitle)
+  Button.groupItem(
+    dataID: DataID.action,
+    content: "Perform action",
+    behaviors: .init { button in
+      print("Button tapped! \(button)")
+    },
+    style: .standard)
+}
+
+// install your group in a view
+group.install(in: view)
+
+// constrain the group like you would a normal subview
+group.constrainToMargins()
+```
+
+As you can see, this is incredibly similar to the other APIs used in Epoxy. One important thing to note is that `install(in: view)` call at the bottom. Both `HGroup` and `VGroup` are written using [`UILayoutGuide`](https://developer.apple.com/documentation/uikit/uilayoutguide) which prevents having large nested view hierarchies. To account for this, we’ve added this `install` method to prevent the user from having to add subviews and the layout guide manually.
+
+Using `HGroup` is almost exactly the same as `VGroup` but the components are now horizontally laid out instead of vertically:
+
+| IconRow |
+| --------- |
+| ![IconRow screenshot](docs/images/IconRow.png) |
+
+```swift
+enum DataID {
+  case icon
+  case title
+}
+
+let group = HGroup(spacing: 8) {
+  ImageView.groupItem(
+    dataID: DataID.icon,
+    content: UIImage(systemName: "person.fill")!,
+    style: .init(size: .init(width: 24, height: 24)))
+  Label.groupItem(
+    dataID: DataID.title,
+    content: "This is an IconRow")
+}
+
+group.install(in: view)
+group.constrainToMargins()
+```
+
+You can learn more about `EpoxyLayoutGroups` in its [wiki entry](https://github.com/airbnb/epoxy-ios/wiki/EpoxyLayoutGruops).
```

Comment:
Might be nice to have a quick comment that VGroups/HGroups can be arbitrarily nested to create advanced component layouts